src/main/kotlin/com/faendir/intellij/gradleVersionCatalogs/kotlin/Accessor.kt

```diff
@@ -15,11 +15,18 @@ private const val BUNDLE_SUPPLIER = "org.gradle.api.internal.catalog.ExternalMod
 private const val PLUGIN_SUPPLIER = "org.gradle.api.internal.catalog.ExternalModuleDependencyFactory.PluginNotationSupplier"
 
 
+@Suppress("SpellCheckingInspection")
 data class Accessor(val element: PsiElement, val id: String, val type: VCElementType) {
     companion object {
         fun find(element: PsiElement): Accessor? {
-            val returnType = element.lastChild.references.map { it.resolve() }.firstIsInstanceOrNull<PsiMethod>()?.returnType ?: return null
-            val segments by lazy { element.text.replace(Regex("\\s+"), "").split(".").drop(1) }
+            val references = if (element.lastChild.references.mapNotNull(PsiReference::resolve).isEmpty()) {
+                element.lastChild.firstChild.references
+            } else element.lastChild.references
```

Comment:
Can you explain why the references code changed?

Reply:
This change is mainly due to the fact that `element.lastChild.references.map { it.resolve() }` is always null when using definitions such as the `map-3dmap` type. because normally `element.lastChild` for `libs.x.y.z` is `z`, but `libs.map.get3dmap()` is `()` . and it is not possible to call `references` to get the result.